seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckConfigUtil.java

```diff
@@ -72,6 +72,16 @@ public static CheckResult checkAtLeastOneExists(Config config, String... params)
         }
     }
 
+    public static boolean isInvalidParam(Config config, String param) {
```

Comment:
It'll be more easy to understand to change to positive method name like 'isValidParam'